--- ./config.inc.org	Tue Jul  9 16:35:31 2002
+++ ./config.inc	Tue Jul  9 16:34:51 2002
@@ -0,0 +1,18 @@
+<?php
+// The database must be set up correctly before using phpToDo.
+// Please check your MySQL manual for more info.
+$mysql_host = 'localhost';
+$mysql_db = 'phptodo';
+$mysql_user = 'phptodo';
+$mysql_passwd = 'phptodo';
+
+// The recommended password should be an MD5 encrypted password.
+// However, you can specify a cleartext password by uncommenting the line.
+
+// MD5 encrypted password.
+$passwd = '098f6bcd4621d373cade4e832627b4f6';
+
+// Cleartext password.
+// $passwd = md5('cleartextpasswordhere');
+?>
+
--- ./index.php.org	Thu Jun 20 22:05:47 2002
+++ ./index.php	Tue Jul  9 16:35:24 2002
@@ -22,22 +22,7 @@
 
 // -----> CONFIGURATION BEGINS HERE <-----
 
-// The database must be set up correctly before using phpToDo.
-// Please check your MySQL manual for more info.
-$mysql_host = 'localhost';
-$mysql_db = 'phptodo';
-$mysql_user = 'phptodo';
-$mysql_passwd = 'phptodo';
-
-// The recommended password should be an MD5 encrypted password.
-// However, you can specify a cleartext password by uncommenting the line.
-
-// MD5 encrypted password.
-$passwd = '098f6bcd4621d373cade4e832627b4f6';
-
-// Cleartext password.
-// $passwd = md5('cleartextpasswordhere');
-
+include "./config.inc"
 // Colors used by CSS can now (0.1.1+) be found in style.css.
 // If you want to modify them, look in the source for the place you want to modify
 // and modify that class in style.css. As of version 0.1.1 phpToDo is XHTML 1.0 with
